typescript-for-haskellers/article.md

```diff
@@ -0,0 +1,726 @@
+# TypeScript for Haskellers
+
+Web app nowadays is mandatory part of every modern application.
+It doesn't matter how small or complex it is.
+From one click apps with pictures converting to photoshop everyone wants fast and easy access to it and Web is where you can give it to your users or even for you with the easiest way.
+
```

Comment:
In some projects, we had a practice that no documentation should be published unless verified with some spellchecker. I also tried to follow it for blog posts, and believe it saved a considerable amount of time.

So I'd really recommend registering in e.g. https://grammarly.com and making the pass over the full text there.